### DISCUSSION TITLE
Give Organizations the Ability to Embed a Tracking Script on Articles

### DISCUSSION THREAD

**Discussion by michael-tharrington** [forem/forem#18308] [Feature Requests]
About a month a half ago, I spoke to an admin of an organization on DEV and gave them a walkthrough of various org abilities + gathered feedback from them to see how we might be able to improve the experience for organizations.

One of the biggest suggestions that they had was to allow organizations access to more statistics on their publication/articles and specifically that they'd **love to be able to embed a tracking script on their articles**.

They pointed out that we have Google Analytics tracking on all of our articles (see screenshot below) and they'd really like it if they could embed their own Google tracking ids so that they can feed this data back into their own system and measure analytics more holistically.

To be clear, this data would not provide them specifics on individual users, but would just extend the analytics that are already available to them in their dashboard (both the org dashboard, e.g. `/dashboard/analytics/org/1` and the specific post analytics, e.g. `/devteam/discussion-and-comment-of-the-week-v13-5fao/stats`) so that they can send these metrics back to the system they're using.

Screenshot:

![Screenshot of Google tracking links when inspecting the code of an article page on DEV](https://user-images.githubusercontent.com/16007075/183689297-7fac18c6-56d5-4ab4-8304-d7225ccba48f.png)

Note: in this screenshot, I point to both UA-##### (Universal Analytics) and G-#### (GA4 Measurement ID) ... both are described in [this resource](https://www.semrush.com/blog/google-analytics-tracking-id/)

**Comment by mikeb-stripe** [comment]
+1 - this functionality would be a game changer for our publishing process at ([Stripe DevRel](https://dev.to/stripe)). We publish loads of articles, but don't have great insight as to how people are finding us, where they're going after they read our docs, and would love to be able to make educated decisions about future content planning.

**Comment by davidbrandt0** [comment]
The suggestion to allow organizations to embed their own Google Analytics tracking IDs into DEV articles is a highly practical improvement that could offer more granular insights into content performance. While DEV already provides basic analytics through the org dashboard and individual post statistics, integrating custom tracking scripts would enable organizations to align DEV data with their internal analytics systems—providing a more holistic view of user engagement and content effectiveness. This kind of feature aligns well with broader trends in digital publishing, where platforms are becoming more open to third-party tracking integrations. For those managing platforms or tools with similar needs, websites like [apknetwork](https://apknetwork.net/) offer valuable insights into apps and utilities that support deeper analytics, tracking, and integration capabilities across various niches.
